CRM/Contract/Form/RapidCreate/AT.php

```diff
@@ -190,12 +190,12 @@ function validate() {
 
   function setDefaults($defaultValues = null, $filter = null) {
 
-    list($defaults['join_date'], $null) = CRM_Utils_Date::setDateDefaults(NULL, 'activityDateTime');
-    list($defaults['start_date'], $null) = CRM_Utils_Date::setDateDefaults(NULL, 'activityDateTime');
+    $defaults['join_date'] = date("Y-m-d H:i:s");
+    $defaults['start_date'] = CRM_Contract_Utils::getDefaultContractChangeDate();
```

Comment:
There's some kind of issue with one of the date fields that causes a JS error on `/civicrm/contract/rapidcreate/at` and `/civicrm/contract/rapidcreate/pl`: `Uncaught Unexpected literal at position 2`. It seems to break the date picker for "Membership start date" (and maybe more).